CATALOG.md

```diff
@@ -580,6 +580,18 @@ Description|http://test-network-function.com/testcases/platform-alteration/isred
 Result Type|normative
 Suggested Remediation|build a new docker image that's based on UBI (redhat universal base image).
 Best Practice Reference|https://connect.redhat.com/sites/default/files/2022-05/Cloud%20Native%20Network%20Function%20Requirements%201-3.pdf Section 5.2
+#### service-mesh
+
+Property|Description
+---|---
+Test Case Name|service-mesh
+Test Case Label|platform-alteration-service-mesh
+Unique ID|http://test-network-function.com/testcases/platform-alteration/service-mesh
+Version|v1.0.0
+Description|http://test-network-function.com/testcases/lifecycle/pod-recreation verifies if service mesh is exist.
```

Comment:
This should be changed from `lifecycle/pod-recreation` --> `platform-alteration/service-mesh`